test/manifest.js

```diff
@@ -174,19 +174,19 @@ test('uses correct base path for each file', async t => {
 	stream.write(createFile({
 		cwd: 'app/',
 		base: 'app/',
-		path: path.join('app', 'foo', 'scriptfoo-d41d8cd98f.js'),
+		path: path.join('app', 'foo', 'scriptfoo-d41d8cd98f.js').replace(/\\/g, '/'),
```

Comment:
This shouldn't be necessary as the path should be normalized in the code. If it is, it's probably a bug somewhere in the code.